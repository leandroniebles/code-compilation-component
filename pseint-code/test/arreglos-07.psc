funcion llenar(a,n por referencia)
  para cada x en a hacer
    x <- n;
    n <- n+1;
  finpara
finfuncion
funcion muestra(nom,a)
  escribir nom,":" sin bajar;
  para cada x en a hacer
    escribir " ",x sin bajar;
  finpara
  escribir "!";
finfuncion
Proceso PruebaArreglos
  dimension a[1],b[2],c[3,3],d[1];
  n<-1;
  llenar(a,n);
  llenar(b,n);
  llenar(c,n);
  muestra("a",a);
  muestra("b",b);
  muestra("c",c);
  para cada x en d hacer
    escribir "ok";
  fin para
FinProceso