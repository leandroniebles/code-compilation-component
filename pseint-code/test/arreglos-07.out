a: 1!
b: 2 3!
c: 4 5 6 7 8 9 10 11 12!
ok
